### PR TITLE
remove ellipses

### PR DESCRIPTION
**Pull request by mirnawong1**
removing ellipses per slack feedback as they were causing errors.

### CODE REVIEW

File: website/docs/docs/cloud/configure-cloud-cli.md [hunk 1]
```diff
@@ -66,9 +66,8 @@ Once you install the dbt Cloud CLI, you need to configure it to connect to a dbt
     ```yaml
     # dbt_project.yml
     name:
-
     version:
-    ...
+    # Your project configs...
```
Comment (mirnawong1) [inline website/docs/docs/cloud/configure-cloud-cli.md:70]: adding this so it's clear to the user that their project info goes here cc @cesher and thank you @runleonarun

**Review by runleonarun — APPROVED** [review]
This looks good to me! Thanks for adding the code comment.

File: website/docs/docs/cloud/configure-cloud-cli.md (hunk 1 above)
Comment (cesher) [reply website/docs/docs/cloud/configure-cloud-cli.md:70]: Looks great, the commend should do it!

**Review by cesher — APPROVED** [review]
Thank you!

**Comment by mirnawong1** [comment]
thank you for the question @runleonarun !

**Pull request merged**